app/knowledge_panels.py

```diff
@@ -39,3 +41,49 @@ def hunger_game_kp(
             ],
         },
     }
+
+
+def data_quality_kp(
+    facet,
+    value: Union[str, None] = None,
+    country: Union[str, None] = None,
+):
+    if facet == "country":
+        path = f"{facet}/{value}"
+        description = f"{value}"
+    else:
+        if country is not None:
+            path = f"country/{country}/{facet}/{value}"
+            description = f"{facet} based for {country}"
+        else:
+            path = f"{facet}/{value}"
+            description = facet
+    url = "https://world.openfoodfacts.org/"
+    source_url = urljoin(url, path)
+    description = f"data-quality issues related to {description}"
+    html = f"{source_url}/data-quality.json"
+    """
+    Parsing data from the url
+    """
+    response_API = requests.get(html)
+    data = response_API.text
+    parse_json = json.loads(data)
+    total_issues = parse_json["count"]
+    # Returns total number of issues
+    tags = parse_json["tags"]
+    first_three = tags[0:3]
+    # Returns First three issues
+
+    return {
+        "data-quality": {
+            "elements": [
+                {
+                    "element_type": "text",
+                    "total_issues": total_issues,
+                    "text_element": first_three,
```

Comment:
change to html, and add a link to see all but not as json (just remove the ".json" at the end of url)

Reply:
Hey @alexgarel , can you please check my current approach for this